lib/stimulus_reflex/channel.rb

```diff
@@ -1,6 +1,18 @@
 # frozen_string_literal: true
 
-class StimulusReflex::Channel < ActionCable::Channel::Base
+module ApplicationCable
+  class Channel < ActionCable::Channel::Base
+    def initialize(connection, identifier, params = {})
+      super
+      application_cable = Rails.root.join("app", "channels", "application_cable", "channel.rb")
+      require application_cable if File.exist?(application_cable)
+    end
+  end
+end
+
+class StimulusReflex::Channel < ApplicationCable::Channel
+  include CableReady::Broadcaster
```

Comment:
Probably I’m not seeing correctly as I’ve got no computer available to try it out, but why did you re-include `CableReady::Broadcaster` here?

Reply:
This significantly predates the recent significant refactoring. It's entirely possible it's included more than it needs to be.

Reply:
You are correct - it doesn't seem to need to be in the channel anymore. Originally, it didn't need to be in the reflex. Things have changed!